java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -213,4 +224,90 @@ public interface BitmapBaseCommands {
      */
     CompletableFuture<Long> bitpos(
             String key, long bit, long start, long end, BitmapIndexType offsetType);
+
+    /**
+     * Reads or modifies the array of bits representing the string that is held at <code>key</code>
+     * based on the specified <code>subCommands</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitfield/">redis.io</a> for details.
+     * @param key The key of the string.
+     * @param subCommands The <code>GET</code> subCommands to be performed on the binary value of the
+     *     string at <code>
+     *     key</code>.<br>
+     *     <ul>
+     *       <li>{@link BitFieldGet} gets the value in {@link Offset} or {@link OffsetMultiplier}
+     *           based on encoding being {@link SignedEncoding} or {@link UnsignedEncoding}.
+     *       <li>{@link BitFieldSet} sets the value in {@link Offset} or {@link OffsetMultiplier}
+     *           based on encoding being {@link SignedEncoding} or {@link UnsignedEncoding}.
+     *       <li>{@link BitFieldIncrby} increases or decreases the value in {@link Offset} or {@link
+     *           OffsetMultiplier} based on encoding being {@link SignedEncoding} or {@link
+     *           UnsignedEncoding}.
+     *       <li>{@link BitFieldOverflow} determines behaviour of {@link BitFieldSet} or {@link
+     *           BitFieldIncrby} when these operations result in under or overflows.
+     *     </ul>
```

Comment:
```suggestion
     *     string at <code>key</code>. The supported subcommands are:<br>
     *     <ul>
     *       <li>{@link BitFieldGet}.
     *       <li>{@link BitFieldSet}.
     *       <li>{@link BitFieldIncrby}.
     *       <li>{@link BitFieldOverflow}.
     *     </ul>
```
All docs that I dropped are already present in the referred classes. If not - please update them.